scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1814,9 +1817,17 @@ then
         DONOR_MYSQL_VERSION=$(parse_sst_info "$sst_file_info_path" sst mysql-version "")
 
         transition_key=$(parse_sst_info "$sst_file_info_path" sst transition-key "")
+
+        # Use the broken key if the donor version is < 5.7.29 and is not 5.7.28-31.31-41.2
+        # In other words, 5.7.28-31.31-41.2 and above will use the key that was sent
+        if ! check_for_version "$DONOR_MYSQL_VERSION" "5.7.29" &&
+           [[ $DONOR_MYSQL_VERSION != "5.7.28-31-31.41.2" ]]; then
+            transition_key="\$transition_key"
```

Comment:
Shouldn't we set here
encrypt_move_options="--transition-key=\\$transition_key --generate-new-master-key"
as well?